packages/server/src/tests/headers/headerSelection.ts

```diff
@@ -0,0 +1,358 @@
+import { HeaderTargeting, HeaderTest, HeaderTestSelection, HeaderVariant } from '@sdc/shared/types';
+
+import { selectVariant } from '../../lib/ab';
+import { audienceMatches, deviceTypeMatches, userIsInTest } from '../../lib/targeting';
+
+import { fetchConfiguredHeaderTestsCached } from './headerTests';
+import { TestVariant } from '../../lib/params';
+
+const modulePathBuilder = header.endpointPathBuilder;
+const moduleName = 'Header';
+
+// --- hardcoded tests
+const nonSupportersTestNonUK: HeaderTest = {
+    name: 'RemoteRrHeaderLinksTest__NonUK',
+    userCohort: 'AllNonSupporters',
+    isOn: true,
+    locations: [
+        'AUDCountries',
+        'Canada',
+        'EURCountries',
+        'NZDCountries',
+        'UnitedStates',
+        'International',
+    ],
+    variants: [
+        {
+            name: 'remote',
+            modulePathBuilder,
+            moduleName,
+            content: {
+                heading: 'Support the Guardian',
+                subheading: 'Available for everyone, funded by readers',
+                primaryCta: {
+                    baseUrl: 'https://support.theguardian.com/contribute',
+                    text: 'Contribute',
+                },
+                secondaryCta: {
+                    baseUrl: 'https://support.theguardian.com/subscribe',
+                    text: 'Subscribe',
+                },
+            },
+        },
+    ],
+};
+
+const nonSupportersTestUK: HeaderTest = {
+    name: 'RemoteRrHeaderLinksTest__UK',
+    userCohort: 'AllNonSupporters',
+    isOn: true,
+    locations: ['GBPCountries'],
+    variants: [
+        {
+            name: 'remote',
+            modulePathBuilder,
+            moduleName,
+            content: {
+                heading: 'Support the Guardian',
+                subheading: 'Available for everyone, funded by readers',
+                primaryCta: {
+                    baseUrl: 'https://support.theguardian.com/subscribe',
+                    text: 'Subscribe',
+                },
+                secondaryCta: {
+                    baseUrl: 'https://support.theguardian.com/contribute',
+                    text: 'Contribute',
+                },
+            },
+        },
+    ],
+};
+
+const supportersTest: HeaderTest = {
+    name: 'header-supporter',
+    userCohort: 'AllExistingSupporters',
+    isOn: true,
+    locations: [
+        'AUDCountries',
+        'Canada',
+        'EURCountries',
+        'GBPCountries',
+        'NZDCountries',
+        'UnitedStates',
+        'International',
+    ],
+    variants: [
+        {
+            name: 'control',
+            modulePathBuilder,
+            moduleName,
+            content: {
+                heading: 'Thank you',
+                subheading: 'Your support powers our independent journalism',
+            },
+        },
+    ],
+};
+
+const baseSignInPromptTest: Omit<HeaderTest, 'name' | 'variants'> = {
+    userCohort: 'Everyone',
```

Comment:
should it be targeting supporters only?

Reply:
I don't think so, if we knew they were a supporter, then there would be no need to prompt them to sign in to recieve the benefits of their support.

Thing we talked about this a bit on chat actually. Does [this](https://chat.google.com/dm/7imLi4AAAAE/-VArS-w1RLQ) link work for you? (might be specific to me)